File: dropwizard-auth/src/main/java/com/lgior/resources/HelloWorldResource.java
package com.lgior.resources;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/hello-world")
@Produces(MediaType.TEXT_PLAIN)
public class HelloWorldResource {

	@GET
	public String sayHello() {
		return "hola!";
	}
}

File: dropwizard-auth/src/main/java/com/lgior/resources/HelloWorldResource.kt
package com.lgior.resources

import javax.annotation.Resource
import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType

@Path("/hello-world")
@Produces(MediaType.TEXT_PLAIN)
class HelloWorldResource {

	@GET
	fun sayHello():String {
		return "Hola"
	}
}